packages/astro/src/default/pages/index.astro

```diff
@@ -3,10 +3,9 @@ import { getTutorial } from '../utils/content';
 
 const tutorial = await getTutorial();
 
-const parts = Object.values(tutorial.parts);
-const part = parts[0];
-const chapter = part.chapters[1];
-const lesson = chapter.lessons[1];
```

Comment:
Why was this 1-based?

Reply:
The folder were expected to start at `1`. We now instead pick the first folder that comes up. It could start at anything: 0, 4, ... or be explicitly defined. 

So this is a new tiny feature! :smiley: